kudos-board-ui/src/components/App/App.jsx

```diff
@@ -32,6 +32,9 @@ function AppContent() {
       <div>
         <BrowserRouter>
           <Routes>
+            {/* 
+            Curious about /homepage vs /homepage/:id
```

Comment:
I'm curious about the reasoning to not have homepage be "/homepage" while the board page is "/homepage/:id".